docs/src/content/sections/pricing.md

```diff
@@ -3,109 +3,110 @@ title: "Choose a pricing plan best suited <em>for you</em>"
+        feature: License management
     price:
       yearly:
-        amount: FREE
-        period: ""
+        amount: 0
+        period: per month
     cta:
       enable: true
-      label: Sign up for a free trial
-      site: teams
-      link: "/signup/trial"
+      label: Download now
+      site: custom
+      link: "https://apps.apple.com/app/apple-store/id1504940162?pt=117264678&ct=pricing-table-free&mt=8"
+      class: "plausible-event-name=App+Store+Install+Free"
   - enable: true
-    title: Individual
-    description: Perfect for solo developers.
+    title: Personal
+    description: Ideal for solo developers—enjoy every Pro feature with an in-app purchase
     price_prefix: "€"
     color: "#1d1d1f"
     features:
       - enable: true
         included: true
-        feature: Pro access to RocketSim.
+        feature: Pro access to RocketSim
       - enable: true
         included: false
-        feature: Team build insights.
+        feature: Team build insights
       - enable: true
         included: false
-        feature: User management.
+        feature: User management
       - enable: true
         included: false
-        feature: License management.
+        feature: License management
     price:
       yearly:
-        amount: 50
-        period: per year
+        amount: 5
+        period: per month
     cta:
       enable: true
       label: Download now
       site: custom
-      link: "https://apps.apple.com/app/apple-store/id1504940162?pt=117264678&ct=website-header&mt=8"
-      class: "plausible-event-name=App+Store+Install"
+      link: "https://apps.apple.com/app/apple-store/id1504940162?pt=117264678&ct=pricing-table-personal&mt=8"
+      class: "plausible-event-name=App+Store+Install+Personal"
   - enable: true
     title: Teams
-    description: Perfect for teams with multiple developers.
+    description: Perfect for teams with multiple developers
     price_prefix: "€"
     color: "#1d1d1f"
     features:
       - enable: true
         included: true
-        feature: Pro access to RocketSim.
+        feature: Pro access to RocketSim
       - enable: true
         included: true
-        feature: Team build insights.
+        feature: Team build insights
       - enable: true
         included: true
-        feature: User management.
+        feature: User management
       - enable: true
         included: true
-        feature: License management.
+        feature: License management
     price:
       yearly:
-        amount: 120
-        period: "per year"
+        amount: 10
+        period: "per month"
     cta:
       enable: true
-      label: Subscribe
+      label: Start 14-day Trial
       site: custom
-      link: "/contact" # TODO: link to Stripe
+      link: "/signup/trial"
   - enable: false
     title: Enterprise
-    description: For larger enterprises with complex requirements.
-    price_prefix: ""
+    description: For larger enterprises with complex requirements
```

Comment:
Question:

Since we don't provide most of the features listed here yet. Should we somehow display a "Coming soon" message? Or should we handle this once we get our first sales contact through this tier?

Reply:
The latter! I don't want to stop anyone from reaching out and get those leads in. That way we can also prioritize better what they need. Secondly, they can already start using the app outside of the App Store while we work on the other features.